src/api/index.js

```diff
@@ -119,7 +116,7 @@ export async function getFavorites() {
 
 export async function addMentorToFavorites(mentorId) {
   const {_id: userId} = await getCurrentUser();
-  const res = await makeApiCall(`${paths.USERS}/${userId}/favorites/${mentorId}`, {}, 'POST');
+  const res = await makeApiCall(`${paths.USERS}/${userId}/favorites1/${mentorId}`, {}, 'POST');
```

Comment:
I think there is a typo here

Reply:
You're quick :) I tested request that failed..